regression/scripts/VTOL/mission_Tiltwing_CRM.py

```diff
@@ -0,0 +1,454 @@
+    segment.process.finalize.post_process.stability = SUAVE.Methods.skip      
+        
+    
+    # add to misison
+    mission.append_segment(segment)     
+                
+    # ------------------------------------------------------------------
+    #   First Cruise Segment: Constant Acceleration, Constant Altitude
+    # ------------------------------------------------------------------
+    
+    segment = Segments.Cruise.Constant_Speed_Constant_Altitude(base_segment)
+    segment.tag = "Cruise"
+    
+    segment.analyses.extend(analyses.cruise)
+    
+    segment.altitude  = 1000.0 * Units.ft
+    segment.air_speed = 110.   * Units['mph']
+    segment.distance  = 30.    * Units.miles                       
+    
+    segment.state.unknowns.propeller_power_coefficient = 0.03 * ones_row(1)
+    segment.state.unknowns.throttle                    = 0.60 * ones_row(1)
+    
+    segment.process.iterate.unknowns.network  = vehicle.propulsors.propulsor.unpack_unknowns
+    segment.process.iterate.residuals.network = vehicle.propulsors.propulsor.residuals    
+    segment.process.iterate.conditions.stability    = SUAVE.Methods.skip
+    segment.process.finalize.post_process.stability = SUAVE.Methods.skip      
+        
+    
+    # add to misison
+    mission.append_segment(segment)     
+                
+    # ------------------------------------------------------------------
+    #   First Descent Segment: Constant Acceleration, Constant Altitude
+    # ------------------------------------------------------------------
+    
+    segment = Segments.Climb.Linear_Speed_Constant_Rate(base_segment)
+    segment.tag = "Descent"
+    
+    segment.analyses.extend(analyses.cruise)
+    segment.climb_rate       = -600. * Units['ft/min']
+    segment.air_speed_start  = 110.   * Units['mph']
+    segment.air_speed_end    = 85.   * Units['mph']
+    segment.altitude_start   = 1000.0 * Units.ft
+    segment.altitude_end     = 40.0 * Units.ft
+    
+    segment.process.iterate.unknowns.network  = vehicle.propulsors.propulsor.unpack_unknowns
+    segment.process.iterate.residuals.network = vehicle.propulsors.propulsor.residuals    
+    segment.process.iterate.conditions.stability    = SUAVE.Methods.skip
+    segment.process.finalize.post_process.stability = SUAVE.Methods.skip      
+        
+    
+    # add to misison
+    mission.append_segment(segment)     
+     
+    # ------------------------------------------------------------------
+    #   Descent Segment: Constant Speed, Constant Rate
+    # ------------------------------------------------------------------
+
+    segment = Segments.Hover.Descent(base_segment)
+    segment.tag = "Arrival"
+
+    segment.analyses.extend( analyses.hover_descent )
+
+    segment.altitude_start  = 40.0  * Units.ft
+    segment.altitude_end    = 0.  * Units.ft
+    segment.descent_rate    = 300. * Units['ft/min']
+    
+    #segment.state.unknowns.throttle                         = 0.5 * ones_row(1)
```

Comment:
Do these comments need to stay?

Reply:
No, it doesn't, I will remove all of them.